app/src/main/res/menu/login.xml

```diff
@@ -13,15 +13,13 @@
 <menu xmlns:android="http://schemas.android.com/apk/res/android">
 
   <!-- TODO: Figure out why applying padding to @style/ActionButton doesn't work. -->
-  <!-- TODO/i18n -->
   <item
       android:id="@+id/action_new_user"
       android:showAsAction="ifRoom"
-      android:title="&#xa0;New User&#xa0;"/>
-  <!-- TODO/i18n -->
+      android:title="@string/title_add_new_user"/>
```

Comment:
You didn't add this to english translation and in French isn't it to long? Wouldn't it be better if instead of *Ajouter un nouvel utilisateur* it was just *nouvel utilisateur*?
![device-2015-10-08-080336](https://cloud.githubusercontent.com/assets/1710823/10364740/5defa6aa-6d94-11e5-9a85-4d138a6d7983.png)